.clang-format

```
@@ -0,0 +1,18 @@
+IndentWidth: 4
+Language: Cpp
+AlignAfterOpenBracket: Align
+SortIncludes: false
+ColumnLimit: 104
```

```suggestion
ColumnLimit: 120
```
I suggest a col limit of 120, looking through several files it looks like a lot of functions either with several args or a lot of indentation are being broken into multiple lines too readily

---

Yeah the 104 limit (inherited from SM64 decomp) is a bit too cozy; 120 works a lot better